index.html

```diff
@@ -3173,13 +3163,19 @@ <h3><code>LinkedResource</code> Definition</h3>
 						<tr>
 							<td>
 								<code>
-									<dfn>alternative</dfn>
+									<dfn>alternate</dfn>
 								</code>
 							</td>
-							<td>A reference to a reformulation of the resource in an alternative format. OPTIONAL</td>
-							<td>One or more valid URL strings&#160;[[!url]].</td>
 							<td>
-								<a href="#value-array">Array</a> of <a href="#value-url">URLs</a>
+								<p>References to one or more reformulation(s) of the resource in alternative formats.</p>
+								<p>When specified, <code>encodingFormat</code> indicates the format of the reformulation.</p>
+								<p>OPTIONAL</p>
+							</td>
+							<td>
+								<p>One or more instances of <a href="#publication-link-def"><code>LinkedResource</code></a> objects.</p>
+							</td>
+							<td>
+								<a href="#value-array">Array</a> of <a href="#value-link">Links</a>
 							</td>
 							<td> (None) </td>
 						</tr>
```

Comment:
I think an example for this property should be added to the spec, to highlight the role of alternate, mainly if a full array is used.

Reply:
I added one